R/utils.R

```diff
@@ -283,3 +283,19 @@
     
     taxa_tab
 }
+
+################################################################################
+# internal wrappers for agglomerateByRank/mergeRows
+.merge_features <- function(x, merge.by, ..., agglomerateTree = FALSE) {
```

Comment:
is `agglomerateTree` necessary here? Can it be passed similarly to other arguments with `...`?

Reply:
ack